source/adapters/cuda/enqueue.cpp

```diff
@@ -843,23 +843,25 @@ UR_APIEXPORT ur_result_t UR_APICALL urEnqueueMemBufferFill(
   }
 }
 
-static size_t imageElementByteSize(CUDA_ARRAY_DESCRIPTOR ArrayDesc) {
+static ur_result_t imageElementByteSize(CUDA_ARRAY_DESCRIPTOR ArrayDesc,
+                                        int *Size) {
   switch (ArrayDesc.Format) {
```

Comment:
Do you need `break`?
https://godbolt.org/z/P766Pvaf1

Reply:
Fixed this, thanks.